stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/viewmodel/ChannelListViewModel.kt

```diff
@@ -77,19 +80,19 @@ public class ChannelListViewModel(
     private val memberLimit: Int = 30,
     private val chatEventHandlerFactory: ChatEventHandlerFactory = ChatEventHandlerFactory(),
     private val chatClient: ChatClient = ChatClient.instance(),
-    private val globalState: GlobalState = chatClient.globalState
+    private val globalState: GlobalState = chatClient.globalState,
 ) : ViewModel() {
 
     /**
      * Represents the current state containing channel list
      * information that is a product of multiple sources.
      */
-    private val stateMerger = MediatorLiveData<State>()
+    private val stateFlow = MutableStateFlow(INITIAL_STATE)
```

Comment:
Could this be renamed to something like `channelListStateFlow`? Since we are using flows naming a variable `stateFlow` makes it harder to understand. Or maybe adding the data type would help with readability? Now it is hard to determine what this state is for and what kind of data is stored.

Reply:
Yes, the naming is confusing, you are correct.

Ideally, I would want to ditch the Flow suffix completely as I think it is superfluous, but then we would have naming collisions and I'm not sure if  the `_` prefix is something we want to widely use.  We do use in it some places, but not in others. 

I agree that `channelListStateFlow` is at least slightly less confusing.